src/components/PollOption.vue

```diff
@@ -87,6 +95,7 @@ export default {
 
 .option-number {
   background-color: var(--option-color);
+  color: v-bind(contrastingTextColor);
```

Comment:
Falls du dir das noch nicht über den Weg gelaufen sein sollte: Das ist https://vuejs.org/api/sfc-css-features.html#v-bind-in-css

Ich glaube das ist meine erste Verwendung davon, sonst setze ich auch eher auf inline Styles.

Reply:
Sehr nice, kannte ich tatsächlich noch nicht. Dass sind echt die kleinen, feinen Unterschiede der API von jedem Framework. Sehr nice, dass das geht.